apps/omg_watcher/test/omg_watcher/exit_processor/piggyback_test.exs

```diff
@@ -314,7 +333,7 @@ defmodule OMG.Watcher.ExitProcessor.PiggybackTest do
       tx_blknum = 3000
 
       # 2. transaction which spends that piggybacked output
-      comp = TestHelper.create_recovered([{tx_blknum, 0, 0, alice}], [])
+      comp = TestHelper.create_recovered([{tx_blknum, 0, 0, alice}], [{alice, @eth, 1}])
```

Comment:
why did we have to start adding outputs?

Reply:
see description, no-output txs are illegal now